cacti/requirements-dev.txt

```diff
@@ -1 +1,3 @@
+docker==3.7.1
+rrdtool==0.1.14
```

Comment:
Should this be in `requirements.in` instead ?

How this integration worked before without `rrdtool` in  https://github.com/DataDog/integrations-core/blob/8479b5a8c66d1031f0c8575d1e5b8c55b99bb4b7/cacti/requirements.in#L1 ?

Reply:
See https://github.com/DataDog/integrations-core/pull/3712
The lib requires manual installation